packages/prop-house-wrapper/src/builders.ts

```diff
@@ -149,11 +137,65 @@ export class Vote extends Signable {
       proposalId: this.proposalId,
       weight: this.weight,
       communityAddress: this.communityAddress,
-      signatureState: this.signatureState,
+      blockHeight: this.blockHeight,
     };
   }
 }
 
+/**
+ * Manages single vote submission with multi-vote signatures.
+ *
+ * To prevent users form signing each vote, we group together all votes and sign one payload.
+ * This one signature is then used to submit each vote to the backend.
+ */
+export class SignableVotes extends Signable {
+  constructor(public readonly votes: Vote[]) {
+    super();
+  }
+
+  /**
+   * The signed payload for a `vote` using a supplied signature
+   */
+  async presignedPayload(
+    vote: Vote,
+    signer: Signer | Wallet,
+    jsonPayload: string,
+    signature: string,
+  ) {
+    const address = await signer.getAddress();
+    return {
+      signedData: {
+        message: Buffer.from(jsonPayload).toString('base64'),
+        signature,
+        signer: address,
```

Comment:
Doesn't really matter for this PR, but want to point out that `signedData.signer` is unnecessary.